week1_KimGaYeon/src/App.jsx

```diff
@@ -0,0 +1,104 @@
+// import { useState } from 'react'
+import './App.css'
+import IconBtn from './components/iconBtn/iconBtn'
+import TabBtn from './components/TabBtn/TabBtn';
+import Card from './components/card/card';
+import { FaBell, FaSearch, FaThumbsUp, FaClock, FaWifi, FaEllipsisV} from 'react-icons/fa';
+import velogLogo from './assets/velog.svg';
+import Dropdown from './components/Dropdown/Dropdown';
+import { useState } from 'react';
+import cardData from './data.js';
+
+function App() {
+  const selectOptions = ['오늘', '이번 주', '이번 달', '올해'];
+  const meatballOptions = ['공지사항','태그 목록', '서비스 정책', 'Slack'];
+  const [isDropdownVisible, setIsDropdownVisible] = useState(false); // 드롭다운 보이기/숨기기 상태
+  const [isMeatballVisible, setIsMeatballVisible] = useState(false); 
+
+  const toggleDropdown = () => {
+    setIsDropdownVisible(!isDropdownVisible); // 드롭다운 상태 토글
+    // console.log(isDropdownVisible);
+  };
+
+  const toggleMeatball = () => {
+    setIsMeatballVisible(!isMeatballVisible); // 드롭다운 상태 토글
+    // console.log(isDropdownVisible);
+  };
+
+
+  return (
+    <>
+    <header>
+      <a href="javascript:location.reload()">
+        <img src={velogLogo} alt="velog 로고" className='logo'></img>
```

Comment:
이렇게 javascript: scheme을 사용하신 이유가 궁금합니다! React 환경에서는 `onClick` 이벤트를 통해서 `window.location.reload()`를 충분히 호출할 수 있고, 이렇게 작성하는것이 html 태그에서 javascript: scheme을 사용하는 방식보다 더 적합하다고 생각합니다.
<img width="577" alt="image" src="https://github.com/user-attachments/assets/33ab5e6b-e2cc-41c7-8264-9aba0d4246e5">
콘솔에서도 warning 메세지가 출력되고 있기 때문에 지양하면 좋을 것 같습니다.
일반적인 링크 동작과 달라서 접근성에 문제가 생길 수도 있을 것 같아요!!

실제로 MDN에서도 지양을 권고하고 있는데요, 코드를 수정해보는 것은 어떨까요?

> Warning: Using javascript: URLs on the web is discouraged as it may lead to execution of arbitrary code, similar to the ramifications of using [eval()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval). It may also reduce [accessibility](https://developer.mozilla.org/en-US/docs/Glossary/Accessibility) because it deviates from normal link behavior.

https://developer.mozilla.org/en-US/docs/Web/URI/Schemes/javascript

```jsx
onClick={() => window.location.reload()}
```